src/API/OnboardingTasks.php

```diff
@@ -36,6 +36,17 @@ class OnboardingTasks extends \WC_REST_Data_Controller {
 	 */
 	protected $rest_base = 'onboarding/tasks';
 
+	/**
+	 * Duration to milisecond mapping.
+	 *
+	 * @var string
+	 */
+	protected $duration_to_ms = array(
+		'day'  => 24 * 60 * 60 * 1000,
```

Comment:
We have constants like `DAY_IN_SECONDS` that may be useful here instead of a new variable.

Reply:
So far I decided to keep the mapping since I find it useful in accessing this in this class, but I'm now utilizing these constants. Thanks for the heads up 👍🏻